### PR TITLE
[ADF-3934]  PeoplCloudComponent - add dynamic placeholder

### PR DESCRIPTION
**Pull request by SilviuCPopa**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://issues.alfresco.com/jira/browse/ADF-3934

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by eromano** [comment]
@mauriziovitale @SilviuCPopa  do we need a title inside this component? will not be better to let the developer choose the context where integrate this component?

**Comment by mauriziovitale** [comment]
> @mauriziovitale @SilviuCPopa do we need a title inside this component? will not be better to let the developer choose the context where integrate this component?

We need a way to be able to override the label. The APA is an example
Screenshot 2019-04-04 at 16 25 30

File: lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <form>
 <mat-form-field class="adf-people-cloud">
-    <mat-label id="assignee-id">{{'ADF_TASK_LIST.START_TASK.FORM.LABEL.ASSIGNEE' | translate}}</mat-label>
+    <mat-label id="assignee-id">{{ title | translate }}</mat-label>
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.html]: can we have a generic id name?

File: lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.spec.ts [hunk 2]
```diff
@@ -145,6 +145,25 @@ describe('PeopleCloudComponent', () => {
                 expect(errorMessage.textContent).toContain('ADF_CLOUD_START_TASK.ERROR.MESSAGE');
             });
         }));
+
+        it('should populate placeholder when title is present', async(() => {
+            component.title = 'ADF_TASK_LIST.START_TASK.FORM.LABEL.ASSIGNEE';
```
Comment (mauriziovitale) [inline lib/process-services-cloud/src/lib/task/start-task/components/people-cloud/people-cloud.component.spec.ts]: can we use a fake title?

**Pull request merged**